ci_framework/roles/cifmw_cephadm/tasks/rgw.yml

```diff
@@ -29,6 +29,20 @@
     _hosts: "{{ _hosts|default([]) + [ hostvars[item][ceph_hostname_var] ] }}"
   loop: "{{ groups['edpm'] }}"
 
+- name: Get and save TLS certificate for rgw spec if provided
+  when: cifmw_cephadm_certificate | length > 0 and
+        cifmw_cephadm_key | length > 0
+  become: true
```

Comment:
do we need `become: true` here?

Reply:
yeah it is not needed, i might have added during my experiments.
I will remove it in my next patchset.